doc/services/pm/device.rst

```diff
@@ -20,90 +27,161 @@ in power saving mode. This method allows saving power even when the CPU is
 active. The components that use the devices need to be power aware and should
 be able to make decisions related to managing device power.
 
-In this method, the SOC interface can enter CPU or SOC power states quickly when
-:code:`pm_system_suspend()` gets called. This is because it does not need to
-spend time doing device power management if the devices are already put in the
-appropriate power state by the application or component managing the devices.
+When using this type of device power management, the kernel can change CPU
+power states quickly when :c:func:`pm_system_suspend()` gets called. This is
+because it does not need to spend time doing device power management if the
+devices are already put in the appropriate power state by the application or
+component managing the devices.
+
+For more information, see :ref:`pm-device-runtime`.
+
+.. _pm-device-system-pm:
 
 System Power Management
 ***********************
 
-In this method device power management is mostly done inside
-:code:`pm_system_suspend()` along with entering a CPU or SOC power state.
+When using this type, device power management is mostly done inside
+:c:func:`pm_system_suspend()` along with entering a CPU or SOC power state.
 
-If a decision to enter a lower power state is made, the implementation would enter it
-only after checking if the devices are not in the middle of a hardware
-transaction that cannot be interrupted. This method can be used in
-implementations where the applications and components using devices are not
-expected to be power aware and do not implement runtime device power management.
+If a decision to enter a CPU lower power state is made, the power management
+subsystem will suspend devices before changing state. The subsystem takes care
+of suspending devices following their initialization order, ensuring that
+possible dependencies between them are satisfied. As soon as the CPU wakes up
+from a sleep state, devices are resumed in the opposite order that they were
+suspended.
 
-.. image:: images/central_method.svg
-   :align: center
+.. note::
+
+   When using :ref:`pm-system`, device transitions can be run from the idle thread.
+   As functions in this context cannot block, transitions that intend to use blocking
+   APIs **must** check whether they can do so with :c:func:`k_can_yield`.
 
-This method can also be used to emulate a hardware feature supported by some
-SOCs which triggers automatic entry to a lower power state when all devices are idle.
-Refer to `Busy Status Indication`_ to see how to indicate whether a device is busy
-or idle.
+This type of device power management can be useful when the application is not
+power aware and does not implement runtime device power management.
+
+.. note::
+
+    When using this type of device power managment, the CPU will only enter
+    a low power state only if no device is in the middle of a hardware
+    transaction that cannot be interrupted.
```

Comment:
I think it's worth mentioning that runtime is preferred over this approach, and that there is an option to disable it now when using runtime PM: `CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE` (enabled by default if `CONFIG_PM_DEVICE_RUNTIME=y`.